modules/redshift-loader/src/main/scala/com/snowplowanalytics/snowplow/loader/redshift/Redshift.scala

```diff
@@ -231,6 +231,10 @@ object Redshift {
                 throw new IllegalStateException("Redshift Loader does not use EventsCopyToTempTable statement")
               case _: Statement.EventsCopyFromTempTable =>
                 throw new IllegalStateException("Redshift Loader does not use EventsCopyFromTempTable statement")
+              case Statement.VacuumEvents =>
+                throw new IllegalStateException("Redshift Loader does not use vacuum events table statement")
+              case Statement.VacuumManifest =>
+                throw new IllegalStateException("Redshift Loader does not use vacuum manifest statement")
```

Comment:
I guess we'll implement this in a future release for Redshift?

Reply:
Of cause.